pkg/report/output/output.go

```diff
@@ -22,99 +24,94 @@ import (
 
 var ErrUndefinedFormat = errors.New("undefined output format")
 
-func GetOutput(
+func GetData(
 	report globaltypes.Report,
 	config settings.Config,
 	baseBranchFindings *basebranchfindings.Findings,
-) (*types.Output[any], error) {
-	var err error
-	var output types.GenericOutput
+) (*types.ReportData, error) {
+	sendToCloud := false
+
+	data := &types.ReportData{}
+	// add detectors
+	err := detectors.AddReportData(data, report, config)
+	if config.Report.Report == flag.ReportDetectors {
+		return data, err
+	}
+
+	if config.Report.Report == flag.ReportDataFlow {
+		err = GetDataflow(data, report, config, false)
+		return data, err
+	}
+
+	// add dataflow to data for internal use
+	if err = GetDataflow(data, report, config, true); err != nil {
+		return data, err
+	}
```

Comment:
We could use a single call here:
```suggestion
	if err = GetDataflow(data, report, config, config.Report.Report != flag.ReportDataFlow); err != nil {
		return data, err
	}
```